int8 command
int16 angle
float32 a